### PR TITLE
Add new JavaScript question

### PR DESCRIPTION
**Pull request by amehi0index**
Checklist:
- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

**Comment by jdwilkin4** [comment]
Hi @amehi0index ! 

Right now your forked copy of the Developer Quiz Site is 12 commits behind the current main branch.
Please update your copy of the repo so you have the most up to date version of the main branch. 
Then you can update your branch and resolve the merge conflict. 
Thanks!

File: src/data/full-quiz.js [hunk 1]
```diff
@@ -1,4 +1,13 @@
 const fullQuiz = [
+  {
+    Question: "Which comparison operator is used to express strict equality, meaning that two compared values have the same type and equal value?",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:3]: For the question, I would like to shorten it and mention JavaScript.

Proposed fix: Which comparison operator is used to express strict equality in JavaScript?

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Hi @amehi0index ! 

Thank you for your quiz question.
I resolved the merge conflict for this branch but you will still need to update your main branch since it is 12 commits behind the FCC main branch. 

Please take a look at the requested change below. 👍

File: src/data/full-quiz.js (hunk 1 above)
Comment (amehi0index) [reply src/data/full-quiz.js:3]: Sounds good

**Comment by jdwilkin4** [comment]
Hi @amehi0index ! 

You still have this open PR.
If you would like to add this question to the site please review those suggested changes otherwise you can go ahead and close this PR.
Thanks! 👍

**Pull request closed without merging**